docs/docs/migrate/keptnapp/asserts/keptnapp-migrated.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: lifecycle.keptn.sh/v1beta1  # changed API version of the resource
+kind: KeptnApp
+metadata:
+  name: "some-keptn-app"
+  namespace: "my-app-ns"
+  labels:
+    my-custom-label: customValue
+    app.kubernetes.io/managed-by: keptn # added annotation
+spec:
+  version: "1.2.3"
```

Comment:
the version string will usually be some hash string which is derived from the contained workloads, or, in the case of a single service application, the version will be the same as the version of the workload (in this case 0.2.7)

Reply:
true, what do you suggest then here ? if any workoload will be updated, new KeptnAppVersion will be re-generated, therefore we need to set the version here to the hash of the workloads. Maybe a small 3rd party script to generate the hash might be useful here ?

Reply:
resolved, as not action is required here. If the workload of the app will be modified, the AppVersion will be updated automatically, as the `managed-by` label is in place